zulip_bots/zulip_bots/bots/joinme/doc.md

```diff
@@ -0,0 +1,30 @@
+# Joinme bot
+
+The Joinme bot posts a link to a Joinme call when being mentioned.
+
+## Setup
+
+1. See [here](https://developer.join.me/docs) for setting up an account and
+   registering the app.
+2. open `bots/joinme/joinme.conf` and change the values of key, secret and the
+   callback url to the ones you created.
+3. See [here](https://zulipchat.com/api/running-bots#running-a-bot) for description
+   on how to run a bot.
```

Comment:
@roberthoenig: Didn't we have a macro for this?

Reply:
I think we did, in the Giphy bot. However, I remember someone changing it back to plain text during GCI: https://github.com/zulip/python-zulip-api/pull/241 We'll do a sweep on all bots in a month or so anyway, so I think this should be fine for now.

Reply:
@roberthoenig: Cool, thanks Robert! :)